src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards v1.0</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+
+      <img class="card__image"
+          src="images\imac.jpeg"
+          alt="iMac 27 inch"
+      >
+
+      <div class="card__title">
+        <div class="titleWrapper">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
```

Comment:
use semantic tags for text

```suggestion
        <h3 class="titleWrapper">
          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
        </h3>
```